### PR TITLE
this pull request is for the day 1 week two exercise.

### PR DESCRIPTION
**Pull request by Drew246**
I will be giving myself a code review of a previous admissions project.

### CODE REVIEW

**Review by Drew246 — COMMENTED** [review]
Please always include the HTML boilerplate. This is an essential part of ALL HTML documents.
You can read more about it here.
(https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/#:~:text=According%20to%20Wikipedia%2C,all%20of%20your%20HTML%20pages.)

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <body>
             <head>
                 <h1 id="title"> Welcome
-                    <p id="description"> this is our survey form
+                    <p id="description"> This is our survey form.
 
 <form id="survey-form">
     <input id="name"required>
```
Comment (Drew246) [inline index.html:10]: Please note the <head>element is used to define the head portion of the document which contains information related to the document. The <head> tag contains other head elements such as <title>, <meta>, <link>, <style> <link> etc.
The correct tag would have been. <main> tag instead.

**Review by Drew246 — COMMENTED** [review]
Please note the element is used to define the head portion of the document which contains information related to the document. The tag contains other head elements such as <title>, , , <style> etc.
The correct tag would have been. <main>

tag instead.

**Review by Drew246 — COMMENTED** [review]
Good job so far but i've noticed the incorrect use of the <head> tag. The <head> tag in HTML is used to define the head portion of the document which contains information related to the document. The <head> tag contains other head elements such as <title>, <meta>, <link>, <style> <link> etc.
please instead make use of the <main> tag for the content of your page.

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Drew246, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Clear and polite comments ✅


## Required Changes ♻️
- You've done a great job so far. However, the project requirements state that you should use Githubs inline comments feature and add at least two inline comments to your project. Please update your project to include this requirement

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
         <body>
             <head>
```
Comment (Drew246) [inline index.html:5]: Good job so far but i've noticed the incorrect use of the tag. The tag in HTML is used to define the head portion of the document which contains information related to the document. The tag contains other head elements such as <title>, , , <style> etc.
please instead make use of the <main> tag for the content of your page.

Comment (Drew246) [inline index.html:5]: Please always include the HTML boilerplate. This is an essential part of ALL HTML documents.
You can read more about it here.
(https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/#:~:text=According%20to%20Wikipedia%2C,all%20of%20your%20HTML%20pages.)

**Review by Drew246 — COMMENTED** [review]
Please review hhe required changes below :recycle:

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @Drew246   👋

Your project is complete! ✔️✔  🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**